mitosheet/src/mito/components/footer/SheetTabContextMenu.tsx

```diff
@@ -1,27 +1,21 @@
 // Copyright (c) Saga Inc.
 
 import React, { useEffect } from 'react';
-import { MitoAPI,  getRandomId } from '../../api/api';
-import { DFSource, GraphDataDict, GraphID, SheetData, UIState } from '../../types';
+import { MitoAPI } from '../../api/api';
+import { DFSource, GraphDataArray, GraphID, SheetData, UIState } from '../../types';
 import Dropdown from '../elements/Dropdown';
 import DropdownItem from '../elements/DropdownItem';
 import DropdownSectionSeperator from '../elements/DropdownSectionSeperator';
 import { ModalEnum } from '../modals/modals';
-import { getDefaultGraphParams } from '../taskpanes/Graph/graphUtils';
 import { TaskpaneType } from '../taskpanes/taskpanes';
 
 /*
     Helper function for finding all of the graph tab names
     that are created from a given sheet index
 */
-export const getGraphTabNamesAndIDsFromSheetIndex = (sheetIndex: number, graphDataDict: GraphDataDict): ({graphTabName: string, graphID: GraphID})[] => {
-    // Filter to only grapsh with the sheetIndex, and then get a list of the graph tab names
-    const filteredGraphDataJSON: GraphDataDict = Object.fromEntries(Object.entries(graphDataDict || {}).filter(([, graphData]) => {
```

Comment:
I removed this code but I actually wanted to check in about what it originally did - I'm kind of confused by the comment?

Reply:
This is meant to be used to find 
```
        const dependantGraphTabNamesAndIDs = getGraphTabNamesAndIDsFromSheetIndex(props.sheetIndex, props.graphDataArray)
```

We def still need this. This is how graphs based on sheets get deleted when those sheets get deleted.